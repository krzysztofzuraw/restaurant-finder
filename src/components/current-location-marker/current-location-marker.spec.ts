import { getCurrentLocationMarker } from '.';

describe('CurrentLocationMarker', () => {
  it('should render correctly', () => {
    const marker = getCurrentLocationMarker();
    expect(marker).toMatchSnapshot();
  });
});
