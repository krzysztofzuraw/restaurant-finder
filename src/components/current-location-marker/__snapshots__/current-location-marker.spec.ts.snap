// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CurrentLocationMarker should render correctly 1`] = `
.emotion-0 {
  width: 11px;
  height: 11px;
  border-radius: 50%;
  background: rgb(36,112,216);
  cursor: pointer;
  box-shadow: 0 0 0 rgba(36,112,216,0.4);
  -webkit-animation: animation-1pz3hvn 2s infinite;
  animation: animation-1pz3hvn 2s infinite;
}

.emotion-0:hover {
  -webkit-animation: none;
  animation: none;
}

<div
  class="emotion-0"
/>
`;
